### PR TITLE
Move contributor

### PR DESCRIPTION
**Pull request by azime**
Move contributor

### CODE REVIEW

File: kirin/rabbitmq_handler.py [hunk 1]
```diff
@@ -86,8 +88,16 @@ def callback(body, message):
             log.info('getting a request: {}'.format(task))
             if task.action != task_pb2.LOAD_REALTIME or not task.load_realtime:
                 return
-
-            feed = convert_to_gtfsrt(RealTimeUpdate.all(task.load_realtime.contributors))
+            begin_date = None
+            end_date = None
+            if task.load_realtime.begin_date:
```
Comment (pbougue) [inline kirin/rabbitmq_handler.py]: you have to use `hasattr(task.load_realtime, "begin_date")` same after

File: kirin/core/model.py [hunk 2]
```diff
@@ -145,12 +146,25 @@ def __init__(self, vj=None):
...
     @classmethod
     def find_by_dated_vj(cls, navitia_trip_id, vj_circulation_date):
         return cls.query.join(VehicleJourney).filter(VehicleJourney.navitia_trip_id == navitia_trip_id,
                                               VehicleJourney.circulation_date == vj_circulation_date).first()
 
+    @classmethod
+    def find_by_contributor_period(cls, contributors, start_date=None, end_date=None):
+        query = cls.query.filter(cls.contributor.in_(contributors))
+        if start_date:
+            query = query.filter("vehicle_journey_1.circulation_date >= '{start_date}'".format(start_date=start_date))
```
Comment (pbougue) [inline kirin/core/model.py]: line length

File: kirin/core/populate_pb.py [hunk 3]
```diff
@@ -43,19 +43,15 @@ def to_posix_time(date_time):
...
     feed.header.timestamp = to_posix_time(datetime.datetime.utcnow())
 
-    for real_time_update in real_time_updates:
-        for trip_update in real_time_update.trip_updates:
-            contributor = None
-            if real_time_update.contributor:
-                contributor = real_time_update.contributor
-            fill_entity(feed.entity.add(), trip_update, contributor)
+    for trip_update in trip_updates:
+        fill_entity(feed.entity.add(), trip_update)
```
Comment (pbougue) [inline kirin/core/populate_pb.py]: No check for contributor?
My guess is we should still check for that (which implies looking in realtime_update)

Comment (pbougue) [inline kirin/core/populate_pb.py]: Ok, probably my bad :)

**Comment by azime** [comment]
corrections

**Pull request merged**